### PR TITLE
Fixed Swift Quickstart issues

### PR DESCRIPTION
**Pull request by Widcket**
This PR fixes a series of issues in the Swift Quickstart, from deprecated method calls to formatting mishaps and typos.

See the list of issues addressed in [this spreadsheet](https://docs.google.com/spreadsheets/d/1bWrIhLP29DXz_l5aps8icdILXYxw4EOQooYoWRxXZ5k/edit?usp=sharing).

### CODE REVIEW

File: articles/quickstart/native/ios-swift/03-user-sessions.md [hunk 1]
```diff
@@ -155,7 +165,7 @@ You will need to update the original login scopes to include `read:current_user`
 
 ### Audience
 
-You will also need to change your audience to the [API Audience identifier](https://manage.auth0.com/#/apis) for the Management API. 
+You will also need to change your audience to the [API Audience identifier](https://manage.auth0.com/#/apis) for the Management API.
```
Comment (joshcanhelp) [inline articles/quickstart/native/ios-swift/03-user-sessions.md]:
```suggestion
You will also need to change your audience to the [API Identifier](https://manage.auth0.com/#/apis) for the Management API.
```

File: articles/quickstart/native/ios-swift/03-user-sessions.md [hunk 2]
```diff
@@ -187,10 +197,13 @@ Auth0
...
 
 ```swift
+// ProfileViewController.swift
+
 let accessToken = ... // You will need the accessToken from your credentials instance 'credentials.accessToken'
 let profile = ... // the Profile instance you obtained accessing the `/userinfo` endpoint.
+
 Auth0
-    .users(token: "user-scoped access token")
+    .users(token: accessToken)
```
Comment (joshcanhelp) [inline articles/quickstart/native/ios-swift/03-user-sessions.md:206]: The one problem with changing this is that the "user-scoped" message is lost (though this is much more clear overall). Maybe just a code comment that mentions the access token needs to be issued on behalf of the user.

Comment (joshcanhelp) [reply articles/quickstart/native/ios-swift/03-user-sessions.md:206]: One more below

**Review by joshcanhelp — CHANGES_REQUESTED** [review]
Looks great ... nice eye for detail here. There were a few tiny things here but otherwise looks great. LMK if you have questions.

File: articles/quickstart/native/ios-swift/03-user-sessions.md (hunk 2 above)
Comment (Widcket) [reply articles/quickstart/native/ios-swift/03-user-sessions.md:206]: I've changed the constant definition to `let accessToken = ... // the user-scoped accessToken from your credentials instance 'credentials.accessToken'`.

**Review by joshcanhelp — APPROVED** [review]

**Pull request merged**